lie/lie/options.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Any, Dict
+from dataclasses import dataclass
+
+import torch
+
+
+def _CHECK_DTYPE_SUPPORTED(dtype):
+    if dtype not in [torch.float32, torch.float64]:
+        raise ValueError(
+            f"Unsupported data type {dtype}. "
+            "Theseus only supports 32- and 64-bit tensors."
+        )
+
+
+@dataclass
+class _TorchLieOptions:
```

Comment:
Minor: wondering if 'options' is the least ambiguous name for the file/class/functions. It sounds more like selecting between some alternatives for some workflow.

Any reason these need to be outside of `constant.py`? If not, we could just use for instance `class _TorchLieConstants`, `set_global_constants`, etc and host everything in that same file.

Reply:
My thinking was that constants should refer to things that we really never want to change, unless need to make an update to the library (e.g., the value of PI, EPS for most tests, etc.), while "options" are things users can change depending on their preferences. 

I guess these Lie group tolerances fall somewhat close to being "constants" according to the definition above, but I figured having flexibility for this wouldn't hurt. And the functionality is now in place for other things.

Reply:
That makes sense to separate out the true 'constants'. Maybe tolerance / eps / params are clearer terms than options?

Reply:
Maybe `params` is better. To be clear, I'm intending for these to be more general than just defining EPS, it's just happens to be the only configuration settings that we have at the moment.

Reply:
Sounds good. Let's go with `params` then if  it's worth the hassle to make the changes.